tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -134,6 +135,20 @@ def montecarlo_radial1d(
     # Condition for Checking if RPacket Tracking is enabled
     if montecarlo_configuration.RPACKET_TRACKING:
         runner.rpacket_tracker = rpacket_trackers
+        runner.rpacket_tracker_df = pd.DataFrame(
+            [
+                {
+                    "index": tracker.index,
+                    "status": tracker.status,
+                    "r": tracker.r,
+                    "nu": tracker.nu,
+                    "mu": tracker.mu,
+                    "energy": tracker.energy,
+                    "shell_id": tracker.shell_id,
+                }
+                for tracker in rpacket_trackers
+            ]
+        )
```

Comment:
This approach seems ok to me. Have you benchmarked it against other methods suggested by @wkerzendorf?

Reply:
this approach makes a dataframe of lists and not a dataframe that contains individual entries